panther_hardware_interfaces/include/panther_hardware_interfaces/panther_system.hpp

```diff
@@ -124,15 +132,19 @@ class PantherSystem : public hardware_interface::SystemInterface
   unsigned max_roboteq_initialization_attempts_ = 2;
   unsigned max_roboteq_activation_attempts_ = 2;
 
-  // SDO error can happen also during setting safety stop (it may be not necessary to use attempts
-  // once we have GPIO controller)
-  unsigned max_safety_stop_attempts_ = 20;
-
   rclcpp::Logger logger_{rclcpp::get_logger("PantherSystem")};
   rclcpp::Clock steady_clock_{RCL_STEADY_TIME};
 
   std::shared_ptr<RoboteqErrorFilter> roboteq_error_filter_;
 
+  float panther_version_;
+
+  std::atomic_bool e_stop_ = true;
+
+  std::atomic_bool last_commands_zero_ = false;
+
+  std::mutex motor_controller_write_mtx_;
+
```

Comment:
```suggestion
  std::atomic_bool e_stop_ = true;
  std::atomic_bool last_commands_zero_ = false;
  std::mutex motor_controller_write_mtx_;

```
I think this can be grupped